v1/content.yaml

```diff
@@ -180,7 +180,11 @@ paths:
 
   /html/{title}:
     x-route-filters:
-      - path: ./lib/revision_table_access_check_filter.js
+      #- path: ./lib/revision_table_access_check_filter.js
+      #  options:
+      #    redirect_cache_control: '{{options.purged_cache_control}}'
+      #    attach_body_to_redirect: true
+      - path: ./lib/access_check_filter.js
```

Comment:
See the commit summary. I enabled this for these two end points so that we can test this, but am proposing to actually revert to revision_table_access_check before merging this PR, so that we can gradually phase in the revision table / back-fill restrictions.